sentry-failure/src/lib.rs

```diff
@@ -49,6 +50,33 @@ fn fail_typename<F: Fail + ?Sized>(f: &F) -> (Option<String>, String) {
     }
 }
 
+#[derive(Default)]
+pub struct FailureIntegration;
```

Comment:
Could give all Integrations a `new` constructor and favor that over using `default` when instantiating? See [`C-CTOR`](https://rust-lang.github.io/api-guidelines/predictability.html#constructors-are-static-inherent-methods-c-ctor):

> The name new should generally be used for the primary method of instantiating a type.